extensions/extensions-helm/aissemble-airflow-chart/README.md

```diff
@@ -48,7 +48,7 @@ but with updated default values.
 | env  | Airflow environment variables | &emsp;- name: KRAUSENING_BASE <br/>&emsp;&emsp;value: /opt/airflow/config/ <br/>&emsp;- name: KAFKA_BOOTSTRAP_SERVER <br/>&emsp;&emsp;value: kafka-cluster:9093 <br/>&emsp;- name: PYTHONUNBUFFERED <br/>&emsp;&emsp;value: "1" <br/>&emsp;- name: GIT_PYTHON_REFRESH <br/>&emsp;&emsp;value: quiet |
 | volumeMounts | Volume mounts for airflow | &emsp;- mountPath: /tmp/model <br/>&emsp;&emsp;name: model <br/>&emsp;- mountPath: /notebooks/boms <br/>&emsp;&emsp;name: boms-notebook                                                                                                                                                             |
 | volumes | Volumes for airflow | &emsp;- name: model <br/>&emsp;&emsp;persistentVolumeClaim: <br/>&emsp;&emsp;&emsp;claimName: model <br/>&emsp;- name: boms-notebook <br/>&emsp;&emsp;persistentVolumeClaim: <br/>&emsp;&emsp;&emsp;claimName: boms-notebook                                                                        |
-
+| namespaceOverride         | The kubernetes namespace to deploy to                                                                           | No                   |
```

Comment:
A: I think we could better convey what's happening with the description. Maybe something like:
_Namespace to deploy to instead of the Helm release namespace._ Also I think the default is more correctly `.Release.Namespace` instead of none.